internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -301,6 +303,52 @@ var Manifest = &appsv1.DaemonSet{
 						//	},
 						// },
 					},
+					{
+						Name:            "log-enricher",
+						Image:           "paulinhu/kube-audit-log-enricher",
+						ImagePullPolicy: v1.PullAlways,
+						VolumeMounts: []v1.VolumeMount{
+							{
+								Name:      "host-devkmsg-volume",
+								MountPath: KmsgDropDirectory,
+								ReadOnly:  true,
+							},
+						},
+						SecurityContext: &v1.SecurityContext{
+							// /dev/kmsg is being used as kernel messages source in order to
+							// support the widest range of distros. However, access to
+							// devices requires the container to run as privileged.
+							Privileged:             &truly,
```

Comment:
I could not find much, so can you explain the "why"? Which kind of privileged do we need beside having the device available within the container?

Reply:
The problem is that (TMK) Kubernetes does not support things such as `--device` that some container run-times have which allows for access to specific devices without the need to run as `privileged`. Therefore, in order to support access to devices you need to run `privileged`.

Trying to simply map the volume into the container and run as root without `privileged` results in `Operation not permitted`, which is fair to be honest.

I already have a few ideas on how to circumvent this - like we did with the non-root enabler. It's coming on follow-up PRs. :)

Reply:
Awesome, thank you for the wrap-up!